examples/custom_dataset/dataset_test.csv

```diff
@@ -0,0 +1,11 @@
+SMILES,value1,value2
```

Comment:
We may create other new test dataset from QM9 instead of deleting train data, if necessary.